build.gradle

```diff
@@ -108,6 +108,7 @@ subprojects { project ->
             it.useVersion '4.3'
           }
         }
+        force 'com.google.api-client:google-api-client:1.22.0'
```

Comment:
@danielpeach I believe when you rollback the GAE lib version you can remove this line.

@jtk54 Is this correct?

Reply:
Yes, that's correct.